sdk/core/azure-core/inc/azure/core/http/curl/curl_connection.hpp

```diff
@@ -15,9 +15,8 @@
 #include <curl/curl.h>
 #include <string>
 
-namespace Azure { namespace Core { namespace Http {
-
-  namespace Details {
+namespace Azure { namespace Core { namespace Http { namespace Details {
+  namespace CurlConstants {
```

Comment:
Why did we add a new namespace?

Reply:
Because this used to be ::Details within the Http namespace.
Once I move all to Http::Details, I am trying to prevent mixing this constants with other Http constants